src/engraving/types/types.h

```diff
@@ -54,6 +54,8 @@ using system_idx_t = size_t;
 using part_idx_t = size_t;
 using page_idx_t = size_t;
 
+using half_step_t = int8_t;
```

Comment:
I'd prefer this to be named `semitones_t`, since that is completely unambiguous (and less informal)

Reply:
Yes, I agree that "semitone" is a better term. I chose "half step" since we use this term in a lot of places